nbs/tutorial.ipynb

```diff
@@ -191,14 +191,24 @@
     "\n",
```

Comment:
<p>I don't understand this part: we want the cell hidden in the docs, that's why we use #hide.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/174/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
I think ReviewNB might be making this hard to read. In jupyter it should look like:

![image](https://user-images.githubusercontent.com/11458288/84147763-dc1c3680-aa55-11ea-8100-cc9651d33383.png)

and in edit mode:

![image](https://user-images.githubusercontent.com/11458288/84148071-564cbb00-aa56-11ea-800f-96f2580d306e.png)

part of the change I just made means that both markdown and code cells will be hidden, if they contain a `#hide` flag.
In this case, we want the code example with `#hide` to show up in the docs, so we use the HTML entity `&#35;` - because `&#35;hide` won't get picked up by `_re_hide`.
If we're using HTML entities, we need to use pre tags rather that triple back-ticks.

make sense?

Reply:
Got it!